runtime/strategies/match-recipe-by-verb.js

```diff
@@ -111,7 +111,8 @@ export class MatchRecipeByVerb extends Strategy {
               }
               for (let i = 0; i < handle.connections.length; i++) {
                 const candidate = handle.connections[i];
-                if (candidate.particle == particleForReplacing && candidate.name == name) {
+                // TODO candidate.name === name triggers test failures
+                if (candidate.particle === particleForReplacing && candidate.name == name) {
```

Comment:
wow, interesting! which test is failing?

Reply:
These tests fail:

  1) MatchRecipeByVerb
       listens to handle constraints:
       carries handle assignments across verb substitution with generic binding:
       selects the appropriate generic binding when handle assignments carry type information:

I will investigate once this is in.